scripts/in_container/check_environment.sh

```diff
@@ -21,6 +21,15 @@ EXIT_CODE=0
 
 DISABLED_INTEGRATIONS=""
 
+function run_nc() {
+    local host=${1}
+    local port=${2}
+    local ip
+    ip=$(python -c "import socket; print(socket.gethostbyname('${host}'))")
```

Comment:
Could you add a comment here (shorter version of the pr desc) saying why we need this.

Reply:
Yep. Just pushed it.